util/sqlutil/rollback.go

```diff
@@ -0,0 +1,38 @@
+package sqlutil
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"testing"
+
+	"github.com/jackc/pgx/v4"
+	"github.com/target/goalert/util/log"
+)
+
+// Rollback perform a DB rollback function
+func Rollback(ctx context.Context, errMsg string, tx *sql.Tx) {
+	if err := tx.Rollback(); err != nil {
+		if err != sql.ErrTxDone && err != sql.ErrConnDone {
+			log.Log(ctx, fmt.Errorf("tx rollback issue at %s: %v", errMsg, err))
+		}
+	}
+}
+
+// RollbackTest will perform a DB rollback for use only within tests
+func RollbackTest(t *testing.T, errMsg string, tx *sql.Tx) {
```

Comment:
Sure thing.  It all depends on how you want to handle it.  I personally would use `t.Fatal` in this case, because it's still tied to the testing framework in case you want to do anything with it.